subprojects/core/src/main/java/org/gradle/initialization/DefaultSettings.java

```diff
@@ -174,7 +174,6 @@ public void include(Iterable<String> projectPaths) {
     }
 
     @Override
-    @Deprecated
     public void includeFlat(String... projectNames) {
```

Comment:
We should also add `includeFlat(Iterable<String>)` to match `include(Iterable<String>)` added in 7.4.

Reply:
Also, are these `projectNames` or `projectPaths`?

Reply:
It's `projectNames`.

Reply:
> We should also add includeFlat(Iterable<String>) to match include(Iterable<String>) added in 7.4.

Let's handle that separately: https://github.com/gradle/gradle/issues/19167

Reply:
@lptr should I do some more here, or can we proceed with the merge and implement `includeFlat(Iterable)` separately?

Reply:
I've created another PR agains this one: https://github.com/gradle/gradle/pull/19168